gensrc/proto/cloud.proto

```diff
@@ -759,6 +761,8 @@ message GetVersionResponse {
     repeated int64 table_ids = 4;
     repeated int64 partition_ids = 5;
     repeated int64 versions = 6;
+    repeated int64 version_update_times = 7;
```

Comment:
what's the single and multiple update_time for?

Reply:
ditto